src/zvbi/decoder.c

```diff
@@ -528,6 +528,9 @@ vbi_raw_decoder_resize		(vbi_raw_decoder *	rd,
 				 int *			start,
 				 unsigned int *		count)
 {
+// Currently no actions are taken on variable
+// Ignores warning about it 
+#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
```

Comment:
These #pragmas should be used to disable specific errors in known places, but once you pass that point warnings needs to be reenabled. Otherwise we might miss them in places we don't know.

Reply:
Yes, sorry. I've forgotten to use #pragma GCC diagnostic push and pop